src/main.ts

```diff
@@ -630,14 +630,18 @@ class NixInstallerAction {
   }
 
   private async setup_kvm(): Promise<boolean> {
+    const current_user = userInfo();
+    const is_root = current_user.uid === 0;
+    const maybe_sudo = is_root ? "sudo" : "";
```

Comment:
Looks like this is inverted, and uses sudo if our euid is 0.

Reply:
Incredible job, Ana.